release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -72,6 +72,27 @@ For more information, see xref:../installing/installing_aws/installing-aws-vpc.a
 [id="ocp-4-14-web-console"]
 === Web console
 
+[id="ocp-4-14-administrator-perspective"]
+==== Administrator Perspective
+
+With this release, there are several updates to the *Administrator* perspective of the web console. You can:
+
+* Narrow down the list of resources in a list view or search page with exact search capabilities. This will help when you have similarly named resources and fuzzy search doesn't narrow down your search.
+* Provide direct feedback about features and report a bug by clicking the *Help* button on the toolbar and click *Share Feedback* from the drop-down list.
+* Display and hide tooltips in the YAML editor. The tooltips will persist so you don't have to change it every time you navigate to the page.
+
+[id="supported-os-types-cluster"]
+===== Retrival of supported operating systems for each cluster node
+
+With this update, the `console-operator` gathers `nodeOperatingSystems` types for each cluster and creates a list of supported operating system types. That list is then passed to the console so it will only render operators in the operator hub that are supported with that operating system type.
```

Comment:
lets rather avoid implementation details.
```suggestion
Operators in Operator Hub are now being filtered based on the nodes operating system, since cluster can contain heterogenous nodes.
```